README.md

```diff
@@ -345,6 +345,16 @@ be sure to clear your cookies.
 
 **NOTE**: At this time, it's not possible to sign in using this approach.
 
+### Working with UX Mocks
+
+When implementing user interfaces you will need to refer to the
```

Comment:
Sure, I can do that. I didn't have soft wrapping enabled in vim but I just fixed that.